idseq_dag/engine/pipeline_step.py

```diff
@@ -262,3 +266,27 @@ def step_resources(self):
         By default, show link to idseq-dag documentation.
         '''
         return {"IDseq Docs": "https://github.com/chanzuckerberg/idseq-dag/wiki"}
+
+
+class PipelineCountingStep(PipelineStep):
```

Comment:
it would be nice to have some docblock here such as

```
PipelineStep that counts nonunique reads based on back-calculation from cluster sizes 
TSV file emitted by `PipelineStepRunCDHitDup`. Most steps should subclass this class. 
```

Reply:
great, thanks